packages/headless/src/features/facets/range-facets/generic/range-facet-utils.ts

```diff
@@ -20,3 +24,15 @@ export const getAnalyticsActionForToggleRangeFacetSelect = (
     ? logFacetDeselect(payload)
     : logFacetSelect(payload);
 };
+
+export const getInsightAnalyticsActionForToggleRangeFacetSelect = (
```

Comment:
Would be best if it's in its own file correctly scoped to **insight** so things don't get too convoluted